scripts/publish.sh

```diff
@@ -37,20 +58,51 @@ function processCommandLineArgs {
 
 function release {
   BUMP=$(npx -p conventional-changelog-angular -p conventional-recommended-bump -c 'conventional-recommended-bump -p angular')
-  VERSION=$(yarn version --no-git-tag-version --new-version ${1:-$BUMP})
+  # jq to get version from yarn https://jqplay.org/s/LqIS_qy2MBx
+  VERSION=$(yarn version --no-git-tag-version --new-version ${release_type:-$BUMP} --json | jq -r 'select(.data | startswith("New version")).data | split(": ")[1]')
```

Comment:
`yarn version` has different output from `npm version`, this parses their output via JSON to get the proper value. Needs jq -r for "raw output" to get the raw value, otherwise the yarn version below fails.